openedxscorm/storage.py

```diff
@@ -18,7 +17,7 @@ def __init__(self, xblock, bucket, querystring_auth, querystring_expire):
         # No need to serve assets from a custom domain.
         self.custom_domain = None
         super().__init__(
-            bucket=bucket,
+            bucket_name=bucket,
```

Comment:
Can you please also modify the `S3ScormStorage` constructor signature such that it matches the super constructor?

```
def __init__(self, xblock, bucket_name=None, querystring_auth=None, querystring_expire=None):
```